testing/test_step_saving.py

```diff
@@ -0,0 +1,232 @@
+import os
+
+import numpy as np
+from joblib import dump
+from py._path.local import LocalPath
+
+from neuraxle.base import BaseStep, TruncableJoblibStepSaver, NonFittableMixin
+from neuraxle.checkpoints import PickleCheckpointStep
+from neuraxle.pipeline import ResumablePipeline
+
+OUTPUT = "OUTPUT"
+ROOT = 'ResumablePipeline'
+CHECKPOINT = 'checkpoint'
+SOME_STEP_2 = 'some_step2'
+PIPELINE_2 = 'pipeline2'
+SOME_STEP_1 = 'some_step1'
+SOME_STEP_3 = 'some_step3'
```

Comment:
should it instead be a list of constants for the some steps? To loop on them.

Reply:
nope doesn't apply for this test. it is a saved pipeline with a structure